src/handlers/channel/readyPermissions.js

```diff
@@ -1,16 +1,24 @@
 const readyPermissions = async (client) => {
-  console.log(`Logged in as ${client.user.tag}!`);
+  console.log('채널', `Logged in as ${client.user.tag}!`);
+
   try {
     // '스스로 만들기' 카테고리의 ID를 확인합니다.
     const guild = client.guilds.cache.get(process.env.SERVER_ID); // 해당 서버의 ID를 넣어야 합니다.
-    const category = guild.channels.cache.find((ch) => ch.name === '스스로 만들기');
-    if (!category) {
+
+    const limitedCategoriesIds = guild.channels.cache
+      .filter((ch) => ch.name.includes('스스로 만들기'))
+      .map((category) => category.id);
+
+    if (!limitedCategoriesIds) {
```

Comment:
```suggestion
    const isValidCategoriesIds = guild.channels.cache
      .filter((ch) => ch.name.includes('스스로 만들기'))
      .map((category) => category.id);

    if (!isValidCategoriesIds) {
```
p4: '스스로 만들기' 부분을  env 변수로 교체하기에 이런 비슷한 형태로 가면 어떨까요?

Reply:
이거 저도 생각 해보긴 했는데 현재 시점에선 나쁘지 않을거 같아요!